core/frontend/src/components/vehiclesetup/configuration/failsafes/Failsafes.vue

```diff
@@ -0,0 +1,200 @@
+    </v-sheet>
+  </div>
+</template>
+
+<script lang="ts">
+import failsafeCard from '@/components/vehiclesetup/configuration/failsafes/FailsafeCard.vue'
+import autopilot_data from '@/store/autopilot'
+
+import { FailsafeDefinition } from './types'
+
+export default {
+  name: 'FailsafesConfigration',
+  components: {
+    failsafeCard,
+  },
+  data() {
+    return {
+      failsafes: [] as FailsafeDefinition[],
+    }
+  },
+  computed: {
+    params_finished_loaded(): boolean {
+      return autopilot_data.finished_loading
+    },
+  },
+  mounted() {
+    // we fetch the failsafes on mounting because we
+    // need to be in an async context to wait for the import calls
+    this.get_failsafes().then((failsafes) => {
+      this.failsafes = failsafes
+    })
+  },
+  methods: {
+    async get_failsafes(): Promise<FailsafeDefinition[]> {
+      const failsafes: FailsafeDefinition[] = [
+        {
+          name: 'Ground Station Heartbeat Loss',
+          generalDescription: 'The heartbeat failsafe is triggered when the vehicle does not receive '
+          + 'a heartbeat from the GCS within 3 seconds.',
+          image: (await import('@/assets/img/configuration/failsafes/heartbeat.svg')).default as string,
+
+          params: [
+            {
+              replacementTitle: 'Timeout',
+              icon: 'mdi-timer',
+              name: 'FS_GCS_TIMEOUT',
+              optional: true,
+            },
+            {
+              replacementTitle: 'Action',
+              icon: 'mdi-lightning-bolt',
+              name: 'FS_GCS_ENABLE',
+              replacementDescription: '',
+            },
+          ],
+        },
+        {
+          name: 'Pilot Input Loss',
+          generalDescription: 'The pilot input failsafe is triggered when the vehicle does not receive '
+          + 'any pilot input for a given amount of time.',
+          image: (await import('@/assets/img/configuration/failsafes/pilot-input.svg')).default as string,
+          params: [
+            {
+              replacementTitle: 'Action',
+              icon: 'mdi-lightning-bolt',
+              name: 'FS_PILOT_INPUT',
+            },
+            {
+              replacementTitle: 'Timeout',
+              icon: 'mdi-timer-outline',
+              name: 'FS_PILOT_TIMEOUT',
+            },
+          ],
+        },
+        {
+          name: 'Leak detection',
+          generalDescription: 'Triggered when a leak is detected. We recoomend keeping control electronics dry.',
+          image: (await import('@/assets/img/configuration/failsafes/leak.svg')).default as string,
+          params: [
+            {
+              replacementTitle: 'Action',
+              icon: 'mdi-lightning-bolt',
+              name: 'FS_LEAK_ENABLE',
+            },
+            {
+              replacementTitle: 'Leak probe 1 type',
+              name: 'LEAK1_TYPE',
+            },
+            {
+              replacementTitle: 'Leak 1 pin',
+              name: 'LEAK1_PIN',
```

Comment:
```suggestion
              replacementTitle: 'Leak 1 pin',
              icon: `mdi-pin',
              name: 'LEAK1_PIN',
```
Not sure if this section is even staying here, but wherever it goes it may as well make use of the icon 🤷‍♂️